test/e2e/beta/from-import-beta-ui.spec.js

```diff
@@ -26,8 +24,7 @@ describe('Using MetaMask with an existing account', function () {
 
   const testSeedPhrase = 'phrase upgrade clock rough situate wedding elder clever doctor stamp excess tent'
   const testAddress = '0xE18035BF8712672935FDB4e5e431b1a0183d2DFC'
-  const testPrivateKey2 = '14abe6f4aab7f9f626fe981c864d0adeb5685f289ac9270c27b8fd790b4235d6'
-  const tinyDelayMs = 500
+  const testPrivateKey2 = 'f5e21e00974fa60353d003938363db0296542d2d24208e88699536940f18fc80'
```

Comment:
Not sure if this is because of the merge conflict. If not, why is `testPrivateKey2` changed in this PR?

Reply:
`testPrivateKey2` has been changed here because the tests that use it, 'Imports an account with private key', are attempting to import an account which is not already present in the accounts list. Because all of the previous private keys have been given balances for one reason or another in the previous tests, they are part of the accounts list and can't be imported a second time.

Reply:
This change is no longer required now that the network switches at the beginning of the 'Imports an account with private key' test chain